janus_core/calculations/descriptors.py

```diff
@@ -4,7 +4,10 @@
 from typing import Any, Optional
 
 from ase import Atoms
+from ase.calculators.calculator import Calculator
+from ase.calculators.mixing import SumCalculator
 import numpy as np
+from torch_dftd.torch_dftd3_calculator import TorchDFTD3Calculator
```

Comment:
why do we need torch? descriptors should not need it.

Reply:
We don't, and the descriptors don't use it, which is why if the calculator is a `SumCalculator` including `TorchDFTD3Calculator`, `get_descriptors` uses the MLIP calculator only.

If a user already has, for example, a `SinglePoint` object using mace-mp + dispersion for other reasons, but wants to also get the descriptors, it would fail without this.

Reply:
I am not very keen to import an entire package just because of the user... I wonder if makes more sense to check if features that involve sum calculator are involved and error on that...

Reply:
Can we be hacky and get away with a name check? It's not as reliable, but avoids the import. Not sure there're any other ways to avoid it and that one also makes my skin crawl to some extent.

Reply:
Yeah that seems reasonable